Bottom: 2a43fef8ad98fd58ad669dd84f30895e874d2df3
Top:    c0e44cdb1d0acd30c4a4d618f90e42c9dd8acc72
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-01-11 14:12:26 +0800

tools: fix virtio test build failure

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/tools/include/xen/xen.h b/tools/include/xen/xen.h
new file mode 100644
index 0000000..44f993c
--- /dev/null
+++ b/tools/include/xen/xen.h
@@ -0,0 +1,6 @@
+#ifndef _XEN_XEN_H
+#define _XEN_XEN_H
+
+#define xen_domain()		(0)
+
+#endif	/* _XEN_XEN_H */
